lib/Test/mysqld.pm

```diff
@@ -154,14 +155,6 @@ sub setup {
     for my $subdir (qw/etc var tmp/) {
         mkdir $self->base_dir . "/$subdir";
     }
-    # copy data files
```

Comment:
It is better copying data before `mysql_install_db` when using `mysql_install_db` for quick bootstrap.
(I know `mysqld --initialize-insecure` doesn't work if copying here)